backup.md

```diff
@@ -77,26 +78,33 @@ scp /path/to/your/<archivename>.tar.gz admin@your.domain.tld:/home/yunohost.back
 
 Alternatively, a solution can be to install Nextcloud or a similar app and configure it to be able to access files in `/home/yunohost.backup/archives/` from a web browser.
 
-Restoring backups
------------------
+### Workaround to retrieve the backups
+
+<div class="alert alert-danger">
+  <span class="glyphicon glyphicon-exclamation-sign"></span> Since Yunohost version 3.3 and the choice to connect as the `admin` user, there is a [limitation](https://github.com/YunoHost/issues/issues/1266) that prevents straight access to the backups folder `/home/yunohost.backup/archives/`
```

Comment:
I'm thinking you mean 3.4 here ?

Anyway, I was wondering about including a migration (or maybe just use one of the existing migration) to simply change the owner of the backup folder to `admin` instead of root, which should fix the issue ?

Reply:
(c.f. https://github.com/YunoHost/yunohost/pull/613 )

Reply:
> I'm thinking you mean 3.4 here ?

Yes, my bad.

------

Tested the `chown` fix by hand on a jessie VM and it works. With admin user, I can copy/paste with Filezilla

```bash
admin@example:~$ tree /home/yunohost.backup
/home/yunohost.backup
├── archives [error opening dir]
└── tmp
    ├── 20180915-212428.info.json
    └── 20180915-212428.tar.gz

2 directories, 2 files


admin@example:~$ ls -al /home/yunohost.backup
total 16
drwxr-xr-x  4 root  root 4096 Jun 17  2018 .
drwxr-xr-x 17 root  root 4096 Sep 15 21:26 ..
drwxr-x---  2 root  root 4096 Jan 12 17:40 archives
drwxr-x---  2 admin root 4096 Jan 12 17:40 tmp

admin@example:~$ ls -al /home/yunohost.backup/archives/
ls: cannot open directory /home/yunohost.backup/archives/: Permission denied
admin@example:~$ 



admin@example:~$ sudo chown admin:root /home/yunohost.backup/archives/


admin@example:~$ ls -al /home/yunohost.backup/archives/
total 1168896
drwxr-x--- 2 admin root      4096 Jan 12 17:40 .
drwxr-xr-x 4 root  root      4096 Jun 17  2018 ..
-rw------- 1 admin 1007      3427 Jun 17  2018 20180611-192934.info.json
-rw------- 1 admin 1007 592976408 Jun 17  2018 20180611-192934.tar.gz
-rw-r--r-- 1 root  root      1436 Sep 15 21:25 20180915-212428.info.json
-rw-r--r-- 1 root  root 164731292 Sep 15 21:25 20180915-212428.tar.gz
-rw-r--r-- 1 root  root         0 Jan 12 17:39 20190115-000000.info.json
-rw-r--r-- 1 root  root         0 Jan 12 17:39 20190115-000000.tar.gz
-rw-r--r-- 1 root  root       331 Dec 31 12:10 dokuwiki--2-pre-upgrade1.info.json
-rw-r--r-- 1 root  root     23800 Dec 31 12:10 dokuwiki--2-pre-upgrade1.tar.gz
-rw-r--r-- 1 root  root       348 Dec 31 12:10 dokuwiki--3-pre-upgrade2.info.json
-rw-r--r-- 1 root  root   3710604 Dec 31 12:10 dokuwiki--3-pre-upgrade2.tar.gz
-rw-r--r-- 1 root  root       344 Dec 31 12:08 dokuwiki-pre-upgrade2.info.json
-rw-r--r-- 1 root  root 163209554 Dec 31 12:08 dokuwiki-pre-upgrade2.tar.gz
-rw-r--r-- 1 root  root       283 Dec 31 12:11 kanboard-pre-upgrade1.info.json
-rw-r--r-- 1 root  root   6336590 Dec 31 12:11 kanboard-pre-upgrade1.tar.gz
-rw-r--r-- 1 root  root       325 Dec 31 12:12 nextcloud-pre-upgrade1.info.json
-rw-r--r-- 1 root  root 114592837 Dec 31 12:12 nextcloud-pre-upgrade1.tar.gz
-rw-r--r-- 1 root  root       236 Dec 31 12:15 piwigo-pre-upgrade1.info.json
-rw-r--r-- 1 root  root  13012940 Dec 31 12:15 piwigo-pre-upgrade1.tar.gz
-rw-r--r-- 1 root  root       260 Dec 31 12:16 roundcube-pre-upgrade1.info.json
-rw-r--r-- 1 root  root  39603447 Dec 31 12:16 roundcube-pre-upgrade1.tar.gz
-rw-r--r-- 1 root  root       263 Dec 31 12:17 wallabag2-pre-upgrade1.info.json
-rw-r--r-- 1 root  root  97465599 Dec 31 12:17 wallabag2-pre-upgrade1.tar.gz
admin@example:~$ 
```